src/CMakeLists.txt

```diff
@@ -37,12 +37,9 @@ endif (WIN32)
 # generate list of common source files
 #-----------------------------------------------------------------------------
 set (NAPISRC napi.c napiu.c nxstack.c nxstack.h stptok.c  nxdataset.c 
+             napi_fortran_helper.c
```

Comment:
This is partly for Nexus.def - I would need to have two version of this file otherwise (one for with fortran symbols, one for without). However it seemed a little strange to me that you could otherwise create two DLL files with the same ABI_VERSION but a different export list?